src/test/kotlin/test/ktortemplate/core/model/TestModels.kt

```diff
@@ -0,0 +1,49 @@
+@file:UseSerializers(InstantAsStringSerializer::class)
+
+package test.ktortemplate.core.model
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.UseSerializers
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import java.time.Instant
+import java.time.format.DateTimeFormatter
+
+@Serializable
+data class TestInstantStringSerialization(val date: Instant)
+
+object InstantAsStringSerializer : KSerializer<Instant> {
+    override val descriptor: SerialDescriptor = PrimitiveSerialDescriptor("Instant", PrimitiveKind.STRING)
+    override fun serialize(encoder: Encoder, value: Instant) =
+        encoder.encodeString(DateTimeFormatter.ISO_INSTANT.format(value))
+
+    override fun deserialize(decoder: Decoder): Instant = Instant.parse(decoder.decodeString())
+}
+
+@Serializable
+data class TestInstantLongSerialization(@Serializable(with = InstantAsLongSerializer::class) val date: Instant)
```

Comment:
@aanciaes  this is an example of overriding a serializer for a particular class?